MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -582,7 +574,7 @@ private async Task<string> ExtractImageInternal(string inputPath, string contain
                 _ => string.Empty
             };
 
-            var mapArg = imageStreamIndex.HasValue ? (" -map 0:v:" + imageStreamIndex.Value.ToString(CultureInfo.InvariantCulture)) : string.Empty;
+            var mapArg = imageStreamIndex.HasValue ? (" -map 0:" + imageStreamIndex.Value.ToString(CultureInfo.InvariantCulture)) : string.Empty;
```

Comment:
Changing this from `-map 0:v:<index>` to `-map 0:<index>`, we tell `ffmpeg` to consider the `<index>` the absolute stream index, instead of the index in the list of video streams.

This means that we can use the `MediaStream.Index` already stored for the stream instead of having to calculate some other index to pass into `ffmpeg`.